### ISSUE TITLE
Report: Show missing data

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/swoptact#17]
A user specifically requests a quality control report. This will flag missing data on review every few weeks (a Participant is missing a phone number, this Action is missing Participants, etc).  

Again, please optionally include primary keys here.

**Comment by cwebber** [comment]
This should be fairly feasible, will require making a view of its own.  We'll need a sense of how we determine what kind of data is missing or not.

The UI on this could get pretty complex, especially if you want things instant and autocomplete'y; consider that to be an inline_ajax.js level of complexity.  If each selection results in the page reloading, it'll probably be much simpler.

**Comment by cecilia-donnelly** [comment]
@cwebber I think for this it's fine to have each selection result in a page reload.  The goal is just for them to be able to check for missing data.

**Comment by cwebber** [comment]
Okay, I was feeling pretty stuck on this, but @cecilia-donnelly helped by talking it through!

Here's what we discussed.  Here's the "ideal" interface we'd both like:

```Event
| name [X]       | description [X] | date [X] | Organizer [X] |
|----------------+-----------------+----------+---------------|
| Bilbo's_Dinner | A farwell party | ******** | ************* |
| Tea_party_____ | Change places!  | ******** | Mad Hatter    |
```

In this design, each model has its own table, and the ****'s get replaced with red backgrounds, and you can check/uncheck the [X] boxes to filter on which "missing data" things you care to look at or not.  However there is probably a lot of work, so Cecilia came up with a simpler alternative, and we can "eventually iterate" to that table approach if appropriate later, if the client wishes to.  Here's the simpler version:

```
| name           | missing information |
|----------------+---------------------|
| Bilbo's_Dinner | date, organizer     |
| Tea_party__    | date                |
```

We also agreed each model should get its own table.  We discussed adding tab interfaces, or doing pagination, but agreed both of these add complexity for now, and we can deal with complexity later as future iterations!

**Comment by cwebber** [comment]
Okay, based on this, I've implemented most of what we discussed.

All that's left at this point is:
 - Link the name of each item shown to its appropriate admin page
 - Link to the missing report page from the admin homepage

Woooooooooo

**Comment by cwebber** [comment]
So it sounds like search is up for debate as to whether or not it will be merged?  That's fine, though I built my stuff off of the search branch to avoid conflicts, so it will have to be redone to skip search for now.

I'm going got try to finish the actual tasks before I attempt a branch which eschews a search branch basis.

**Comment by cwebber** [comment]
So search is back in, this ismplifies things somewhat.

 - Link to the index page has been added
 - links to each item's edit page has been added

Now to add the rest of the models, which should be a piece of cake.

**Comment by cwebber** [comment]
So I added institutions... I figured events, institutions, participants would be the models desired to be shown on this page, but let me know if anything else should be shown.

Otherwise, I think we're ready to go on this once search is merged!

**Comment by cecilia-donnelly** [comment]
@cwebber yes, I think those are the desired models.  Thank you!

**Comment by cwebber** [comment]
Okay, great!

**Comment by cecilia-donnelly** [comment]
This is merged and deployed, so I'm closing the issue.  If we want to implement the more advanced version we can open a separate issue.
